mlflow/genai/judges/utils.py

```diff
@@ -28,19 +28,77 @@
 
 # "endpoints" is a special case for Databricks model serving endpoints.
 _NATIVE_PROVIDERS = ["openai", "anthropic", "bedrock", "mistral", "endpoints"]
+_LITELLM_PROVIDERS = ["azure", "vertexai", "cohere", "replicate", "groq", "together"]
 
 # Global cache to track model capabilities across function calls
 # Key: model URI (e.g., "openai/gpt-4"), Value: boolean indicating response_format support
 _MODEL_RESPONSE_FORMAT_CAPABILITIES: dict[str, bool] = {}
 
 
+def _check_databricks_agents_installed(error_code: str = INVALID_PARAMETER_VALUE) -> None:
+    """Check if databricks-agents is installed for databricks judge functionality.
+
+    Args:
+        error_code: The MLflow error code to use. Defaults to INVALID_PARAMETER_VALUE.
+
+    Raises:
+        MlflowException: If databricks-agents is not installed.
+    """
+    try:
+        import databricks.agents.evals  # noqa: F401
+    except ImportError:
+        raise MlflowException(
+            f"To use '{_DATABRICKS_DEFAULT_JUDGE_MODEL}' as the judge model, the Databricks "
+            "agents library must be installed. Please install it with: "
+            "`pip install databricks-agents`",
+            error_code=error_code,
+        )
+
+
 def get_default_model() -> str:
     if is_databricks_uri(mlflow.get_tracking_uri()):
         return _DATABRICKS_DEFAULT_JUDGE_MODEL
     else:
         return "openai:/gpt-4.1-mini"
 
 
+def validate_judge_model(model_uri: str) -> None:
+    """
+    Validate that a judge model URI is valid and has required dependencies.
+
+    This function performs early validation at judge construction time to provide
+    fast feedback about configuration issues.
+
+    Args:
+        model_uri: The model URI to validate (e.g., "databricks", "openai:/gpt-4")
+
+    Raises:
+        MlflowException: If the model URI is invalid or required dependencies are missing.
+    """
+    from mlflow.metrics.genai.model_utils import _parse_model_uri
+
+    # Special handling for Databricks default model
+    if model_uri == _DATABRICKS_DEFAULT_JUDGE_MODEL:
+        # Check if databricks-agents is available
+        _check_databricks_agents_installed()
```

Comment:
I think we should always just use `BAD_REQUEST`; no need to parametrize by error code.